packages/@tinacms/schema-tools/src/types/index.ts

```diff
@@ -7,6 +7,8 @@ type Meta = {
   error?: any
 }
 
+export type FilterValue = string[] | string // extend this type if needed
```

Comment:
```suggestion
export type FilterValue = string[] | string 
```

Reply:
i intentionally leave the comment there, this type is extendable when we need to support other datatype in future

Reply:
ah lets give it more info then! 
what its used for, where its used, etc